test/e2e/framework/util.go

```diff
@@ -410,33 +410,6 @@ func CheckTestingNSDeletedExcept(ctx context.Context, c clientset.Interface, ski
 	return fmt.Errorf("Waiting for terminating namespaces to be deleted timed out")
 }
 
-// WaitForServiceEndpointsNum waits until there are EndpointSlices for serviceName
-// containing a total of expectNum endpoints. (If the service is dual-stack, expectNum
-// must count the endpoints of both IP families.)
-//
-// Deprecated: use e2eendpointslice.WaitForEndpointCount or other related functions.
-func WaitForServiceEndpointsNum(ctx context.Context, c clientset.Interface, namespace, serviceName string, expectNum int, interval, timeout time.Duration) error {
```

Comment:
Move this to the other commit; clean up autoscaling in the first commit, then remove WaitForServiceEndpointsNum and countEndpointsSlicesNum in the second one.